app/src/main/java/org/httpsknighthacks/knighthacksandroid/Profile.java

```diff
@@ -0,0 +1,40 @@
+package org.httpsknighthacks.knighthacksandroid;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class Profile extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState){
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_profile);
+
+        Button settingsButton = findViewById(R.id.settings_button);
+        settingsButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                // TODO setContentView(R.layout.activity_settings);
```

Comment:
Calling setContentView multiple times to change the layout of the activity is not a good practice. What you might want to use are Fragments instead, but let's wait to see what the settings portion will look like before doing this.